### ISSUE TITLE
The bert encoder without self-attention mask is right?

### ISSUE THREAD

**Issue by reck1ezz** [bojone/bert4keras#280]
In the method apply_main_layers in Bert, the MultiHeadAttention layer don't use mask to ignore the padding words in the batch, which is same with Keras_Bert library. I think it is wrong!!!!

**Comment by reck1ezz** [comment]
I want to know whether it is a special trick in bert encoder. In the transformers library of pytorch, the input of bert model includes the attention mask.

**Comment by bojone** [comment]
Keras will automatically create masking for zero input due to `mask_zero=True` in `Embedding` layer.

**Comment by reck1ezz** [comment]
Oh, god, thand you very much. I have read the <Understanding masking & padding> in the keras docs, and understand the _keras_mask. Thank you again taking time out to answer my questions.

**Comment by yccckid** [comment]
> Keras will automatically create masking for zero input due to `mask_zero=True` in `Embedding` layer.

But in Keras_Bert, I do not know how to use an Embedding, because I just use load_trained_model_from_checkpoint method, how can I add an Embedding layer? plz
